libs/git_wrapper/Git_wrapper.ml

```diff
@@ -93,6 +93,8 @@ exception Error of string
  *)
 let _git_diff_lines_re = {|@@ -\d*,?\d* \+(?P<lines>\d*,?\d*) @@|}
 let git_diff_lines_re = Pcre_.regexp _git_diff_lines_re
+let _remote_repo_name_re = {|^http.*\/(.*)\.git$|}
```

Comment:
please avoid to use _ as a prefix which is used to disable the unused-variable check.
If it's because you want to show that this constant is not exported, you can use it as a local as in

let report_repo_name_re =
   let re = {|...|} in
   Pcre_.regexp re

Reply:
A leading underscore silences the warnings about the variable being unused.

Reply:
Fair! I forgot _ disables the unused var check